silq/analysis/fit_toolbox.py

```diff
@@ -298,9 +300,13 @@ def add_to_plot(self,
             **{self.sweep_parameter: x_vals_full})
         x_vals_full *= xscale
         y_vals_full *= yscale
-        plot_kwargs = {**self.plot_kwargs, **kwargs}
+
+        # Set default plot kwargs while de-aliasing (e.g. 'lw' -> 'linewidth')
+        # kwargs to prevent duplicate keys
+        kwargs = {**self.default_plot_kwargs,
+                  **cbook.normalize_kwargs(kwargs, mlines.Line2D)}
```

Comment:
Wouldn't you want to apply normalize_kwargs on `{**self.default_plot_kwargs, **kwargs}`?

Reply:
No, that doesn't work. 

For example if you have:
```python
default_plot_kwargs = {'linewidth' : 2}
kwargs = {'lw': 3}

cbook.normalize_kwargs({**default_plot_kwargs, **kwargs}, mlines.Line2D)
```
This raises the following error:
```python-traceback
TypeError: Got both 'lw' and 'linewidth', which are aliases of one another
```

--- 

I could also normalize`self.default_plot_kwargs`, but I would like to think that people won't change the default arguments and pass in clashing kwargs too often...

Reply:
Ah alright, didn't realize that normalize_kwargs couldn't handle that